jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/ReceiverConstraintsMap.kt

```diff
@@ -57,4 +59,9 @@ class ReceiverConstraintsMap {
         }
         return maxSeen
     }
+
+    fun getDebugState() = OrderedJsonObject().apply {
+        put("maxHeight", maxHeight)
+        put("constraints", map)
```

Comment:
Does this leak the map object outside the synchronization?

Reply:
Technically, yes.  I can do `map.toMap()` to make a copy there.